src/cli/lint/internalLints/upgrade/index.ts

```diff
@@ -61,7 +61,6 @@ const fileExtensions = ['js', 'ts'];
 const resolvePatches = async (version: string): Promise<Patches> => {
   for (const extension of fileExtensions) {
     try {
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
```

Comment:
These autofixes make me think that the `@typescript-eslint` rules are not applying correctly now 😞. Probably needs a deeper think.